projects/observability/src/shared/dashboard/data/graphql/table/entity/entity-table-data-source.model.ts

```diff
@@ -129,4 +129,14 @@ export class EntityTableDataSourceModel extends TableDataSourceModel {
       sort: parentRequest.sort
     };
   }
-}
+
+  private buildDefaultSortArg(columns: SpecificationBackedTableColumnDef[]): { key: Specification; direction: TableSortDirection } | undefined {
+
+    const defaultSortColumn = columns.find(column => column.sort);
+    
+    return defaultSortColumn ? {
+      key: defaultSortColumn.specification,
+      direction: defaultSortColumn.sort ?? TableSortDirection.Ascending
+    } : undefined;
+  }
+}
```

Comment:
Don't forget to add a newline at the EOF 😉

Reply:
oh yeah, updated